Control/Parallel/Strategies.hs

```diff
@@ -296,13 +296,26 @@ withStrategy = flip using
 -- | Compose two strategies sequentially.
 -- This is the analogue to function composition on strategies.
 --
--- For any strategies @strat1@, @strat2@, and @strat3@,
+-- > strat2 `dot` strat1 == strat2 . withStrategy strat1
+--
+-- 'dot' is associative:
 --
 -- > (strat1 `dot` strat2) `dot` strat3 == strat1 `dot` (strat2 `dot` strat3)
--- > strat1 `dot` strat1 = strat1
--- > strat1 `dot` r0 == strat1
 --
--- > strat2 `dot` strat1 == strat2 . withStrategy strat1
+-- 'r0', 'rseq', and 'rpar' are all one-sided identities of 'dot':
+--
+-- > strat `dot` r0 == strat
+-- > strat `dot` rseq == strat
+-- > strat `dot` rpar == strat
+--
+-- Furthermore, if @strat@ only forces and sparks computations (that is, it
+-- doesn't actually change any values), then
```

Comment:
This makes it sounds like changing values is an OK thing to do.  How about "Furthermore, since @strat@ should only force and spark compurations, not change any values..."

Reply:
We seem to disagree quite strongly on that point.